service/gitlab/gitlab_mr_discussion.go

```diff
@@ -15,6 +13,8 @@ import (
 	"github.com/reviewdog/reviewdog/service/serviceutil"
 )
 
+var X = "debug"
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported var X should have comment or be unexported